notebooks/tools/cuml_benchmarks.ipynb

```diff
@@ -331,8 +337,8 @@
    "outputs": [],
    "source": [
     "runner = cuml.benchmark.runners.SpeedupComparisonRunner(\n",
-    "    bench_rows=SMALL_ROW_SIZES, \n",
-    "    bench_dims=WIDE_FEATURES,\n",
+    "    bench_rows=TINY_ROW_SIZES, \n",
```

Comment:
Does this make our speedup not look great because it's so small? Maybe a comment in the cell like - "Larger speedups come with more data, but the CPU version then increases run time excessively, so we only compare small datasets here."